### PR TITLE
Update FAPI client

### PR DESCRIPTION
**Pull request by gidsg**
Updates FAPI client, this required the following additional changes:

* Explicitly import `json4s` as this is no longer provided by the CAPI models used by FAPI
* Use `CirceDecoders` to decode JSON into Content model as `parseContent` is no longer provided by the CAPI models used by FAPI
* Change Scanamo version to match the version of Cats used by other dependencies (see comment in `build.sbt` for more details)

CC @Reettaphant @janua @LATaylor-guardian

### CODE REVIEW

File: app/util/ContentUpgrade.scala [hunk 1]
```diff
@@ -46,10 +47,11 @@ object ContentUpgrade {
   def upgradeItem(json: JValue): JValue = {
     Try({
       val jsonString = JsonMethods.compact(JsonMethods.render(json))
-      val maybeCapiContent = JsonParser.parseContent(jsonString)
+      val maybeParsedJson: Option[Json] = parse(jsonString).toOption
```
Comment (gidsg) [inline app/util/ContentUpgrade.scala]: This is a Circe `parse`

**Comment by gidsg** [comment]
This is an updated version of #218

File: build.sbt [hunk 2]
```diff
@@ -73,15 +74,29 @@ libraryDependencies ++= Seq(
...
+    "io.circe" %% "circe-core" % circeVersion,
+    "io.circe" %% "circe-generic" % circeVersion,
+    "io.circe" %% "circe-parser" % circeVersion,
+    "com.gu" %% "scanamo" % "0.7.0",
+
+
     "com.gu" %% "thrift-serializer" % "1.1.0",
     "net.logstash.logback" % "logstash-logback-encoder" % "4.7",
     "org.julienrf" %% "play-json-variants" % "2.0",
+    "org.json4s" % "json4s-native_2.11" % "3.5.0",
```
Comment (LATaylor-guardian) [inline build.sbt]: Use %% here to omit the scala version.

**Comment by LATaylor-guardian** [comment]
Why were the changes needed in `public/src/js/jspm-config.js`?

**Comment by LATaylor-guardian** [comment]
Looks okay from my point of view other than my comments, but it's not a project I have involvement in personally so will default to @janua @Reettaphant

**Comment by gidsg** [comment]
@LATaylor-guardian good spot, think these JS changes were just auto generated when I ran the install script, but I will revert them.

File: app/util/ContentUpgrade.scala [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
-import com.gu.contentapi.json.JsonParser._
 import com.gu.facia.api.utils.{CardStyle, ResolvedMetaData}
 import com.gu.facia.client.models.TrailMetaData
 import org.json4s.JValue
 import org.json4s.JsonAST._
 import org.json4s.JsonDSL._
-import org.json4s.native.JsonMethods
+import org.json4s.jackson.JsonMethods
+import io.circe._
+import io.circe.parser._
```
Comment (janua) [inline app/util/ContentUpgrade.scala]: Is it possible to be more explicit about what you are importing from `circe`, such as `parse`?

**Comment by janua** [comment]
LGTM 👍
You should get a tools team plus one.

**Comment by Reettaphant** [comment]
👍

**Pull request merged**